OIPA/api/transaction/serializers.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework import serializers
+from iati.models import Transaction
+from iati.models import TransactionType
+
+from api.generics.serializers import DynamicFieldsModelSerializer
+from api.organisation.serializers import OrganisationSerializer
+from api.activity.serializers import ActivitySerializer
+from api.activity.serializers import AidTypeSerializer
+from api.activity.serializers import CurrencySerializer
+from api.activity.serializers import DescriptionTypeSerializer
+from api.activity.serializers import FinanceTypeSerializer
+from api.activity.serializers import FlowTypeSerializer
+from api.activity.serializers import TiedStatusSerializer
+
+
+class TransactionTypeSerializer(DynamicFieldsModelSerializer):
+    class Meta:
+        model = TransactionType
+        fields = (
+            'code',
+            'name',
```

Comment:
TransactionType is a reference to a codelist.
We decided to only serialize 'code' in this case and not the rest of the fields